src/libraries/Common/src/System/Security/Cryptography/MLKem.cs

```diff
@@ -1432,8 +1449,6 @@ public static MLKem ImportEncryptedPkcs8PrivateKey(string password, ReadOnlySpan
         /// </remarks>
         public static MLKem ImportFromPem(ReadOnlySpan<char> source)
         {
-            ThrowIfNotSupported();
```

Comment:
Why was this removed?

Reply:
For consistency with the encrypted case. I preferred this because the 99+% case is going to be use on supported platforms so this avoids the double ThrowIfPlatformNotSupport check (since the callbacks use the public Import* methods which also do the check). The previous version may be more readable/obvious though so I'm open to either version as long as it's consistent.

Reply:
Probably a question for @bartonjs.

Reply:
There is no double check, at least not if the method gets enough use to do tiered JIT.  The IsSupported is a `static readonly`, so it'll get const-promoted and either the throw removed or the rest of the method after it.

Unless there's a good reason to remove the throw call, it should remain.